python_modules/dagster/dagster/core/log_manager.py

```diff
@@ -200,6 +210,8 @@ def _log(
         for logger in self._loggers:
             logger.log(level, msg, *args, extra=extra)
 
+        super()._log(level, msg, args)
```

Comment:
should be super()._log(level, msg, args, exc_info=exc_info, extra=extra, stack_info=stack_info)

see comment below for an example of where this behavior comes into play.

Reply:
thanks for pointing this out, didn't notice this nuance! modified this line and the corresponding test.